en-US/firefox-ios.xliff

```diff
@@ -4548,6 +4528,18 @@ Private Tab</target>
       </trans-unit>
     </body>
   </file>
+  <file original="Shared/Supporting Files/en.lproj/Shopping.strings" datatype="plaintext" source-language="en" target-language="en-US">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="14.0.1" build-num="14A400"/>
+    </header>
+    <body>
+      <trans-unit id="Shopping.Sheet.Title.v118" xml:space="preserve">
```

Comment:
@petruSt24: Sorry seems like this fell through the cracks of my review yesterday. Given this is the first time we introduce a string about Fakespot, we should clarify in comments a little bit what Fakespot is.
Also I'm not sure right now what the string refers to, as in, what does it do? Probably good to clarify as well (and/or if you have a screenshot I might share, that would be even better :) )
LMK! thanks